lib/api/core/validation/types/date.js

```diff
@@ -165,38 +170,53 @@ class DateType extends BaseType {
    * @throws {PreconditionError}
    */
   validateFieldSpecification(typeOptions) {
+<<<<<<< HEAD
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Parse error at line 173 column 2:

163: 
164:     return true;
165:   }
166: 
167:   /**
168:    * @param {TypeOptions} typeOptions
169:    * @return {TypeOptions}
170:    * @throws {PreconditionError}
171:    */
172:   validateFieldSpecification(typeOptions) {
173: <<<<<<< HEAD
      ^
174:     if (typeOptions.hasOwnProperty('formats')) {
175:       if (!Array.isArray(typeOptions.formats) || typeOptions.formats.length === 0) {
176:         errorsManager.throw('internal', 'validation', 'invalid_formats_option');
177: =======
178:     if (_.has(typeOptions, 'formats')) {
179:       if (
180:         !Array.isArray(typeOptions.formats)
181:         || typeOptions.formats.length === 0
182:       ) {
183:         throw new PreconditionError('Invalid "formats" option: must be a non-empty array'); [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AParsingError)